### PR TITLE
alpine: update main description

### PR DESCRIPTION
**Pull request by waldyrious**
----

- [_] ~~The page (if new), does not already exist in the repo.~~

- [_] ~~The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.~~

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/alpine.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # alpine
 
-> A easy-to-use Email and Internet News program.
-> With pico/nano inspired interface and support for most modern Email services through IMAP.
+> An email client and Usenet newsgroup program with a pico/nano-inspired interface.
+> Supports most modern Email services through IMAP.
```
Comment (sbrl) [inline pages/linux/alpine.md]: "Email" is capitalised here? That doesn't feel right. it's only people / places that should be capitalised in the middle of a sentence, IIRC.

File: pages/linux/alpine.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
 - Open alpine normally:
 
 `alpine`
 
-- Open alpine directly to the message composition screen to send an email to {{email_address}}:
+- Open alpine directly to the message composition screen to send an email to a given email address:
 
-`alpine {{email_address}}`
+`alpine {{email@address.net}}`
```
Comment (sbrl) [inline pages/linux/alpine.md]: `address` -> `domain` / `example`?

Comment (waldyrious) [reply pages/linux/alpine.md]: Sure.

File: pages/linux/alpine.md (hunk 1 above)
Comment (waldyrious) [reply pages/linux/alpine.md]: That was an overlook -- I meant to lowercase it from the previous wording, but that one slipped. I'll change.

**Comment by waldyrious** [comment]
Hey @sbrl, when I notice that two other maintainers have been recently active, I normally let the third one merge the other one's PRs when I'm the only one who has approved them.

I see the two-maintainer approval bypass (when one of them is the PR author) mostly as a useful exception to prevent the project from slowing down too much.

That way us maintainers are exposed to the same process as everyone else and can better identify weak points or temporary bottlenecks :)

**Comment by sbrl** [comment]
@waldyrious Ah, ok! I'll bear that in mind. So if there are lots of maintainers active, then PRs by maintainers still require 2 different people's approval (merging on the second one)?

**Comment by waldyrious** [comment]
Sure, that's just a suggestion, not a policy or hard-set rule. We always have to balance for example with simple fixes vs. more complete rewritings, etc.

**Comment by agnivade** [comment]
Haha, no worries. It's totally fine @sbrl. Feel free to merge. @waldyrious went crazy and sent a flood of PRs, so it's a bit hard to keep up 😛

**Comment by waldyrious** [comment]
No, no, what happened was I went crazy and became absent for a while (what was I thinking??). These are the PRs I'd have sent in a regular pace during that period :stuck_out_tongue_winking_eye:

<sub>ps - there are probably a handful yet to come.<sub>

**Comment by sbrl** [comment]
Ah, ok! I totally see now :P I might have a small one of my own soon too, actually

**Pull request merged**